_episodes/01-intro.md

```diff
@@ -94,7 +94,7 @@ command doesn't always require an argument or a flag.
 In the second line of the example above, our **command** is `ls`, with a **flag** `-F` and an
 **argument** `/`. Each part is separated by spaces: if you omit the space 
 between `ls` and `-F` the shell will look for a command called `ls-F`, which 
-doesn't exist. Also, capitalization matters: `LS` is different to `ls`. 
+doesn't exist. Also, capitalization can matter: `LS` is usually different to `ls`. 
```

Comment:
Good point - I think we need further explanation here, as including the word *usually* may naturally make learners curious about when capitalization matters.

Reply:
I was trying to avoid overcomplicating the material but correcting the inaccuracy (which causes in-lesson questions like 'but LS isn't different to ls on my computer' from Mac users).

Would 'often' be better than 'usually', or does that suffer the same issue in your mind?

Reply:
What do you think about something like this:

> Also, capitalization can matter: depending on your operating system or its configuration, `LS` may be different from `ls`.

Reply:
It's depends on the filesystem being used, not the operating system - would being a bit more general than 'operating system' be acceptable? e.g. 'Also, capitalization can matter: depending on your system and configuration, LS may be different from ls.'?

Reply:
I favour keeping the current wording, in the interest of simplicity, especially considering that this is at the earliest stages of the shell novice lesson where avoiding unnecessary complications has a very high priority. Generally, there's no need to point learners to every possible quirk / corner case / convenience. Getting them started with simplified, safe and interoperable rules and techniques is a reasonable teaching practice that's frequently applied throughout our lessons, as far as I'm aware.

I'd also argue that paying attention to capitalisation is generally a good idea, even if some shell implementations may somehow not require that. Leading learners to expect / depend on this kind of hidden auto-canonicalisation may well turn out to be a disservice to them when they move on to Python or R.

And finally, from a perhaps rather more advanced perspective I'd also argue that sticking with consistent capitalisation is a good "clean coding" habit.

Reply:
#796 is now closed.